Wrappers/Python/cil/processors/Binner.py

```diff
@@ -0,0 +1,238 @@
+#%%
```

Comment:
Please update the headers as in [here](https://github.com/vais-ral/CCPi-Framework/issues/540#issuecomment-718794402) and remove the python2 imports.